common/changes/office-ui-fabric-react/focus_panel_2019-01-15-18-11.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Setting the aria-modal flag to true on Panel because it's considered modal and currently, focus is going outside the Panel component while Narrator's in scan mode. This alone will not solve this problem fully because we're still waiting on Narrator to support aria-modal.",
```

nit: too much info in the release comment. Maybe consider the comment bellow and remove the word `Fixes` before the issue number in the PR description so you don't close the issue. Leave a comment in the issue about the waiting on Narrator to support it and keep it open for tracking purposes. You decide what to do... just a suggestion 😄 
```suggestion
      "comment": "Panel: setting the aria-modal to true to prevent loosing focus when using Narrator in scan mode.",
```